.travis.yml

```diff
@@ -5,7 +5,9 @@ d:
   - ldc
 
 before_script: git fetch --unshallow
-script: "dub --config=test"
+script:
+  - "dub --config=test"
+  - cd test_package && dub build
```

Comment:
This will no be run when the tests are running on Windows. Either add it to `test.d` or `appveyor.ps1`. If it's added to `test.d` it will be cross-platform and also be tested when running the tests locally.